pkg/scheduler/controller/shoot/reconciler_test.go

```diff
@@ -800,17 +793,8 @@ var _ = Describe("Scheduler_Control", func() {
 			Expect(bestSeed).To(BeNil())
 		})
 
-		It("should fail because it cannot find a seed cluster due to not bootstrapped", func() {
-			seed.Status.Conditions = []gardencorev1beta1.Condition{
-				{
-					Type:   gardencorev1beta1.SeedGardenletReady,
-					Status: gardencorev1beta1.ConditionTrue,
-				},
-				{
-					Type:   gardencorev1beta1.SeedBootstrapped,
-					Status: gardencorev1beta1.ConditionFalse,
-				},
-			}
+		It("should fail because it cannot find a seed cluster never reconciled before", func() {
```

Comment:
```suggestion
		It("should fail because it can only find a seed cluster never reconciled before", func() {
```
or 
```
		It("should fail because it cannot find a seed cluster reconciled at least once before", func() {
```